piqe_ocp_lib/api/resources/ocp_configs.py

```diff
@@ -0,0 +1,80 @@
+from piqe_ocp_lib.api.resources.ocp_base import OcpBase
+from kubernetes.client.rest import ApiException
+import logging
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+
+class OcpConfig(OcpBase):
+    """
+        OcpConfig Class extends OcpBase and encapsulates all methods
+        related to managing Openshift Config related operations.
+        :param kind: (str) kubernetes/openshift resource kind/type
+        :param api_version: (str) kubernetes/openshift api version
+        :param kube_config_file: A kubernetes config file.
+        :return: None
+        """
+
+    def __init__(self,  kind, api_version, kube_config_file=None):
```

Comment:
Isn't kind is 'Config' here? or what is the kind we are expecting here?  Why are we passing api_version here?

Reply:
Yes, kind will be Config here. I made it parameterized so instead of hard coding. So incase if it's different kind type for different resource to get config details, this class should help to provide details.
Reason we pass the api_version is because different resources in openshift have different api_version for config. So if users wants to get config data for different object which might have different api_version, they should be able to use it.